src/bundle/Resources/public/js/alloyeditor/src/toolbars/config/base-table.js

```diff
@@ -1,7 +1,10 @@
 import AlloyEditor from 'alloyeditor';
```

Comment:
Now, this import seems unused.

Reply:
Yes, I checked again now and I think all was now moved from the original PR